GameData/RealismOverhaul/Waterfall_Configs/ReStock/BornholioNuclear/restock_KIWI-A24.cfg

```diff
@@ -0,0 +1,11 @@
+@PART[KIWIA24]:NEEDS[Waterfall,ReStock]:BEFORE[ROWaterfall]
+{
```

Comment:
Why do all the Bornholio Nuclear patches require Restock?  (Is there no waterfall plume for the stock nuclear engine?)

Reply:
Each model (stock, VSR) requires separate configuration due to the nature of Waterfall plumes. I'll make plumes for the VSR models at some point, but I myself won't be making them for stock parts because the artwork quality is not there.

But yes, currently, stock and VSR will use the existing plume.